tests/tf-manifests/rhcs/clusters/rosa-hcp/main.tf

```diff
@@ -93,3 +93,78 @@ resource "rhcs_hcp_default_ingress" "current" {
   cluster          = rhcs_cluster_rosa_hcp.rosa_hcp_cluster.id
```

Comment:
shouldn't we update the `rhcs_hcp_default_ingress` with `var.full_resources` check in `count ` ?